packages/ek-components/src/styles.scss

```diff
@@ -32,13 +32,13 @@ $modal-backdrop-opacity: 0.8;
 $app-body-overflow: scroll;
 
 $theme-colors: (
-  "bundle": $lavender,
-  "audio": $pink,
-  "document": $cyan,
-  "exercise": $teal,
-  "html5": $yellow,
-  "zim": $yellow,
-  "video": $orange
+  "bundle": $gray-300,
+  "audio": $gray-300,
+  "document": $gray-300,
+  "exercise": $gray-300,
+  "html5": $gray-300,
+  "zim": $gray-300,
+  "video": $gray-300
```

Comment:
:+1: I guess it's not worth the effort refactoring this now as design decision to make all the same gray may change.